xbmc/pvr/PVRContextMenus.cpp

```diff
@@ -345,12 +346,17 @@ namespace PVR
 
     bool DeleteRecording::IsVisible(const CFileItem& item) const
     {
+      const std::shared_ptr<CPVRClient> client = CServiceBroker::GetPVRManager().GetClient(item);
+      if (client && !client->GetClientCapabilities().SupportsRecordingsDelete())
```

Comment:
We need to take care that all PVR add-on's capabilities are updated properly. Otherwise they will no longer offer deletion of recordings at GUI level.

Reply:
Yes, of course.

Reply:
Done